xspf2m3u.py

```diff
@@ -121,11 +122,18 @@ def parse_args():
     parser.add_argument('folder')
     parser.add_argument('-Y', '--youtube', action='store_true')
     parser.add_argument('-O', '--outdir')
+    parser.add_argument('-T', '--target')
```

Comment:
I don't think this needs to be optional. I see no harm in always generating extended M3U.

Reply:
OK

Reply:
If we want to download with wget, this option can be useful